.github/workflows/r.yml

```diff
@@ -28,6 +28,7 @@ jobs:
       - name: Install dependencies
         uses: r-lib/actions/setup-r-dependencies@v2
         with:
+          xtra-packages: local::. # Necessary to avoid object usage linter errors.
           cache-version: 2
           needs: check
           pak-version: 'devel'
```

Comment:
```suggestion
          cache-version: 2
          needs: check
          pak-version: 'devel'
          extra-packages: local::. # Necessary to avoid object usage linter errors.
```